docs/reference/commandline/service_create.md

```diff
@@ -312,15 +314,16 @@ volumes in a service:
     <th>Description</th>
   </tr>
   <tr>
-    <td><b>types</b></td>
+    <td><b>type</b></td>
     <td></td>
     <td>
-      <p>The type of mount, can be either <tt>volume</tt>, <tt>bind</tt>, or <tt>tmpfs</tt>. Defaults to <tt>volume</tt> if no type is specified.
+      <p>The type of mount, can be either <tt>volume</tt>, <tt>bind</tt>, <tt>tmpfs</tt>, or <tt>npipe</tt>. Defaults to <tt>volume</tt> if no type is specified.
       <ul>
         <li><tt>volume</tt>: mounts a <a href="https://docs.docker.com/engine/reference/commandline/volume_create/">managed volume</a>
         into the container.</li> <li><tt>bind</tt>:
         bind-mounts a directory or file from the host into the container.</li>
         <li><tt>tmpfs</tt>: mount a tmpfs in the container</li>
+		<li><tt>npipe</tt>: mounts named pide from the host into the container.</li>
       </ul></p>
     </td>
   </tr>
```

Comment:
Could you also update the `required` column at line 332 below https://github.com/docker/cli/pull/1195/files#diff-d4779426535b6679e406cf721c9f9289R332 ?

It currently says;

```
<td>for <tt>type=bind</tt> only></td>
```

Which should now be;

```
<td>for <tt>type=bind</tt> and <tt>type=npipe</tt></td>
```

(Noticed there's a stray `>` there as well)